src/dbms/inmemory/replication_handlers.cpp

```diff
@@ -435,7 +435,7 @@ uint64_t InMemoryReplicationHandlers::ReadAndApplyDelta(storage::InMemoryStorage
         auto vertex = transaction->FindVertex(delta.vertex_add_remove_label.gid, View::NEW);
         if (!vertex)
           throw utils::BasicException("Invalid transaction! Please raise an issue, {}:{}", __FILE__, __LINE__);
-        auto ret = vertex->AddLabel(transaction->NameToLabel(delta.vertex_add_remove_label.label));
+        auto ret = vertex->AddLabel(transaction->NameToLabel(delta.vertex_add_remove_label.label), true);
```

Comment:
Why is here `update_text_index == true` shouldn't that be taken from the config?